flyteidl/protos/flyteidl/core/tasks.proto

```diff
@@ -192,6 +192,8 @@ message ContainerPort {
     // Number of port to expose on the pod's IP address.
     // This must be a valid port number, 0 < x < 65536.
     uint32 container_port = 1;
+    // Name of the port to expose on the pod's IP address.
+    string name = 2;
```

Comment:
what's an example of a port name?

Reply:
K8s allow giving ports names so that services can reference pod ports by name and that can change under the cover...